lib/Container/CoreExtension.php

```diff
@@ -50,12 +50,23 @@ class CoreExtension implements ExtensionInterface
 
     public function getDefaultConfig()
     {
-        return [
+        $config = [
             'autoload' => 'vendor/autoload.php',
             'cwd' => getcwd(),
             'console_dumper_default' => 'indented',
             'cache_dir' => __DIR__ . '/../../cache',
         ];
+
+        $composerConfig = json_decode(
+            file_get_contents(getcwd() . '/composer.json'),
+            true
+        );
+
+        if (isset($composerConfig['config']['vendor-dir'])) {
+            $config['autoload'] = $composerConfig['config']['vendor-dir'] . '/autoload.php';
+        }
```

Comment:
What if we create an `Phpactoe\Config\AutoloadGuesser` class and set the above `autoload` parameter to `null` ?

The `AutoloadGuesser` can be registered as a service and used [here](https://github.com/phpactor/phpactor/pull/209/files#diff-890d76bedc8f8763259ce12c500a3b04R212) if `autoload` is null.